packages/reakit/src/Stack/Stack.ts

```diff
@@ -0,0 +1,33 @@
+import { theme, withProp } from "styled-tools";
+import styled from "../styled";
+import use from "../use";
+import Box, { BoxProps } from "../Box";
+
+export type StackAnchor = "top" | "right" | "bottom" | "left";
+
+export interface StackProps extends BoxProps {
+  anchor?: StackAnchor[];
+}
+
+const ifAnchor = (anchor: StackAnchor, style: string) =>
+  withProp("anchor", anchors => (anchors.includes(anchor) ? style : ""));
+
+const Stack = styled(Box)<StackProps>`
+  position: relative;
+
+  > *:not(:first-child) {
```

Comment:
Is the `*` selector what we want? Will these styles not propagate to all rendered childs (_not only the top level ones_)?